docs/blog/what-to-expect-as-a-speaker.rst

```diff
@@ -0,0 +1,53 @@
+.. post:: Jan 07, 2022
+   :tags: speakers, conferences
+   :author: Kat Stoica
+
+###################################
+Speaking at WTD --- what to expect?
```

Comment:
Since this is the first time we're doing a community-written blog post, I think a notice at the top of the page would be useful, something like "This guest blog post has been written by our community member <name>. We're happy to share experiences from our community. If you'd like to contribute a blog post about your WTD experiences, email us at support@writethedocs.org. "

Reply:
Yep, absolutely, and a shout out to the communit member as well. :+1: